workflow-dev/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: deis-logger-fluentd
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  template:
+    metadata:
+      name: deis-logger-fluentd
+      labels:
+        heritage: deis
+    spec:
+      serviceAccount: deis-logger-fluentd
+      containers:
+      - name: deis-logger-fluentd
+        image: quay.io/deisci/fluentd:canary
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: varlog
+          mountPath: /var/log
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+        env:
+        - name: SYSLOG_HOST
+          value: $(DEIS_LOGGER_SERVICE_HOST)
+        - name: SYSLOG_PORT
+          value: $(DEIS_LOGGER_SERVICE_PORT_TRANSPORT)
+      volumes:
+      - name: varlog
+        hostPath:
+          path: /var/log
+      - name: varlibdockercontainers
+        hostPath:
+          path: /var/lib/docker/containers
```

Comment:
So it seems that right now its pretty uniform but we are finding edge cases that im going to address in another PR (i need to fix it in fluentd first)